### PR TITLE
feat: getsentry/salt@v3005-sentry-pypi for ops

### PR DESCRIPTION
**Pull request by joshuarli**
edit: Just gonna ship this without salt==3005 since there's currently no mechanism to publish to internal pypi (since we'd wanna avoid public registry for these kinds of things), and Anthony has a few designs in mind. For now in ops I'm just gonna have people install a manually built salt wheel.

### CODE REVIEW

File: packages.ini [hunk 1]
```diff
@@ -707,6 +742,7 @@ validate_incorrect_missing_deps = beautifulsoup4
 
 [uritemplate==4.1.1]
 
+[urllib3==1.26.6]
```
Comment (asottile-sentry) [inline packages.ini:745]: I happen to know this version has a vulnerability

Comment (joshuarli) [reply packages.ini:745]: link? couldn't find anything on it on a quick search

**Review by asottile-sentry — APPROVED** [review]

**Review by asottile-sentry — APPROVED** [review]

**Pull request merged**